core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5222MojoDeprecatedParamsTest.java

```diff
@@ -35,7 +35,7 @@
 {
     public MavenITmng5222MojoDeprecatedParamsTest()
     {
-        super( "[3.9.0,)" );
+        super( "[4.0.0-alpha-1,)" );
```

Comment:
Why back to 4 and not with 3.9?

Reply:
When this change will be merged with current fix to Maven master test on 3.9 will fail. 
I will restore it with together cherry pick/port change to 3.9.x branch of Maven core.